code/Control/UserDefinedFormController.php

```diff
@@ -108,6 +108,9 @@ protected function addUserFormsValidatei18n()
      */
     public function index(HTTPRequest $request = null)
     {
+        if ($this->config()->disable_form_content_interpolation) {
+            return [];
```

Comment:
I've not seen this shortcut much in the wild.. This is the equivalent of saying there's no custom data to render?

I think it's much more intuitive / conventional to return `$this` if you don't want to customise the controller

---

Here's the behaviour if your action returns an array: https://github.com/silverstripe/silverstripe-framework/blob/4/src/Control/Controller.php#L286-L291 I don't think this is what we want to be doing

Reply:
Yeah `$this` is more consistent with other SS code. Returning a blank array was done to maintain the `@return array`.